frontend/src/routes/login.lazy.tsx

```diff
@@ -65,18 +66,94 @@ function Login() {
             Log in
           </h1>
           <div className="card flex flex-col gap-3">
-            {data?.errors?.root && (
-              <div style={{ color: "red" }}>{data.errors.root}</div>
-            )}
-            <div>
+            <Form.Root>
+              {data?.errors && (
+                <Form.Message
+                  className="text-red-500 text-sm"
+                  match={"valueMissing"}
+                >
+                  {data.errors.root}
+                </Form.Message>
+              )}
```

Comment:
Do not use a `Form.message` for this error, instead use an Alert component : https://ui.shadcn.com/docs/components/alert#destructive

This is because this message is not linked to a field, it is a global error, and radix will scream at you because a field need to be provided.